oeps/oep-0001.rst

```diff
@@ -289,6 +298,12 @@ ReStructuredText [8] allows for rich markup that is relatively easy to read,
 and can also be rendered into good-looking and functional HTML. OEPs are
 rendered to HTML using Sphinx. An `OEP template`_ can be found in the repo.
 
+OEPs may be discussed in a more convenient format, such as a Google Doc, if
+it is deemed appropriate for the audience and sufficiently open to comment and
+review. The final OEP should be committed as described above with a link to the
```

Comment:
Yeah, I'd like to make sure that's clear, since one of the original goals of the OEP process was to make sure we had a single source of truth for accepted proposals.